### ISSUE TITLE
Diplomas para los trabajadores completando cada planta

### ISSUE THREAD

**Issue by eak24** [AguaClara/APP_Issue_Tracking#1]
En un conversatorio con German, Antonio y yo. Antonio sugerio dar diplomas a los trabajadores de cada planta, en particular, debemos de dar diferente diplomas a cada tipo de trabajo, por ejemplo dar un diploma para los maestros de obra, Albaniles ,   para los supervisors o ingenieros . segun Antonio es muy importante para indicar quien es un experto de la tecnologia AguaClara especialmente cuando otra empresas empieza construir AguaClara plantas.

**Comment by jonschristensen** [comment]
Hmmm sí sería util para ellos y relativamente sencillo. No conozco la costumbre de diplomas en la construcción allá. Tal vez ver que piensan Minty y Santiago, ultimamente la oficina tendrá que imprimir y certificarlos...

**Comment by mintynunez** [comment]
Me gusta la idea! es un incentivo y un detalle muy bonito para los trabajadores por todo su esfuerzo. Puede ser un diploma de agradecimiento o un reconocimiento por su trabajo por su participación en la construcción de la planta AguaClara pero no podemos certificarlos. Solo seria algo que tengan como demostrar su experiencia.

**Comment by eak24** [comment]
@rvl24 te gusta este idea?

**Comment by rvl24** [comment]
Sí, me parece bien. En el mundo de AguaClara siempre intentamos crear un sentido de comunidad, y creo que esto ayudaría con eso.
